src/py/flwr/common/recordset_test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""RecordSet tests."""
+import secrets
+
+import numpy as np
+
+from .parameter import ndarrays_to_parameters, parameters_to_ndarrays
+from .parametersrecord import Array
+from .recordset_utils import (
+    parameters_to_parametersrecord,
+    parametersrecord_to_parameters,
+)
+from .typing import NDArray, NDArrays, Parameters
+
+
+def get_ndarrays() -> NDArrays:
+    """Return list of NumPy arrays."""
+    arr1 = np.array([[1.0, 2.0], [3.0, 4], [5.0, 6.0]])
+    arr2 = np.eye(2, 7, 3)
+
+    return [arr1, arr2]
+
+
+def nparray_to_array(np_array: NDArray) -> Array:
+    """Represent NumPy array as Array."""
+    return Array(
+        np_array.tobytes(),
+        dtype=str(np_array.dtype),
+        stype="np.tobytes",
```

Comment:
good idea! we could default to using the fully qualified module name that someone could use to load / look up this function:

```suggestion
        stype="numpy.ndarray.tobytes",
```